test.cram

```diff
@@ -15,3 +15,38 @@ Test okay file.
 
     $ echo 'This is okay.' > good.txt
     $ $SCSPELL good.txt
+
+
+Test file with --override-dictionary and a fileid mapping entry
+
+    $ cp -a "$TESTDIR/tests" .
+    $ $SCSPELL --override-dictionary "tests/fileidmap/dictionary" \
+    > --relative-to "tests/fileidmap" \
+    > "tests/fileidmap/inputfile.txt" "tests/fileidmap/inputfile2.txt"
+    tests/fileidmap/inputfile.txt:3: 'soem' not found in dictionary (from token 'soem')
+    tests/fileidmap/inputfile2.txt:4: 'soem' not found in dictionary (from token 'soem')
+    [1]
+
+Test fileid manipulations
+
+    $ $SCSPELL --override-dictionary tests/fileidmap/dictionary \
+    > --relative-to tests/fileidmap \
+    > tests/fileidmap/mix*
+    $ $SCSPELL --override-dictionary tests/fileidmap/dictionary \
+    > --relative-to tests/fileidmap \
+    > --merge-fileids tests/fileidmap/mix4.txt tests/fileidmap/mix1.txt
+    $ mv tests/fileidmap/mix2.txt tests/fileidmap/mix22.txt
+    $ $SCSPELL --override-dictionary tests/fileidmap/dictionary \
+    > --relative-to tests/fileidmap \
+    > --rename-file tests/fileidmap/mix2.txt tests/fileidmap/mix22.txt
+    Switching fileid d5641cfe-3574-11e6-a3a6-10ddb1d4c3d5 from mix2.txt to mix22.txt
+    $ $SCSPELL --override-dictionary tests/fileidmap/dictionary \
+    > --relative-to tests/fileidmap \
+    > --delete-file tests/fileidmap/mix5.txt
+    Removing tests/fileidmap/mix5.txt <-> 7f72eb84-3576-11e6-a3a6-10ddb1d4c3d5 mappings
+    $ diff tests/fileidmap/dictionary tests/fileidmap/dictionary.post
```

Comment:
The repo versions are in $TESTDIR/tests.  Earlier in test.cram, I added "cp -a "$TESTDIR/tests ." to have a local copy to modify.  (and so invoking test.cram doesn't modify the git working directory.  (And to make all the commandlines shorter.))

The --merge-fileids and --rename-file and --delete-file invocations modify ./tests/fileidmap/dictionary.  Then the diff compares the result to ./tests/fileidmap/dictionary.post.  The .post file  is unmodified since the copy, and contains the expected result of those operations.  The diff could just as well have been
 ` diff tests/fileidmap/dictionary $TESTDIR/tests/fileidmap/dictionary.post`